### PR TITLE
Feature/shachaf/fix down scale prompt

### PR DESCRIPTION
**Pull request by ShachafIronSource**
fix down-scale target on prompt, and some additional help improvements

### CODE REVIEW

File: spotinst-cli [hunk 1]
```diff
@@ -393,11 +392,13 @@ for group, group_values in groups_to_update.iteritems():
...
             command = "up"
             adjustment = options.scale_up
+            target = int(group_values[2]) + int(adjustment)
         elif options.scale_down is not None:
             command = "down"
             adjustment = options.scale_down
+            target = int(group_values[2]) - int(adjustment)
         # check that this change does not pass the percent_alert_threshold
-        ans = check_change_in_target(group, group_values[2], int(adjustment) + int(group_values[2]), percent_alert_threshold)
+        ans = check_change_in_target(group, group_values[2], target, percent_alert_threshold)
```
Comment (orarnon) [inline spotinst-cli:401]: I see that target is used only a single time, why use it as a variable?

Comment (ShachafIronSource) [reply spotinst-cli:401]: it is called twice, on lines 395, 399. it is differ by the ain order to show the right target amount on the text.

Comment (orarnon) [reply spotinst-cli:401]: It's not called twice, it's two different cases - only one is applicable at a time.
OK, I see your point: up until now, it only upscaled the group, right?

Comment (ShachafIronSource) [reply spotinst-cli:401]: no. the action was correct, but the message about the target to-be changed was wrong.

**Pull request merged**